TappNetwork/home-visiting-app-web
# Cu 3kpbxzz search select

# Details
Search select has been added and the search multi select component have been abstracted into NPM package. In addition, some code cleanup has been done to quiet some linter warnings, and when a user changes their profile, the changes appear immediately with no need for a refresh.

# Steps to Test
1. Run 'npm install' and 'npm run serve'
2. Login with visitor user
3. Go to resource center and test search select in the share resource modal
4. Go to forms page and test the search multi select
5. go to the edit profile page and test the search multi selects
6. submit a profile change, the changes should now be immediately visible on the view profile page without a refresh.

# Tasks linked
[3kpbxzz](https://app.clickup.com/t/3kpbxzz)
